### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by alexcrow1974** [chris2511/xca#433] [Show and tell]
I don't really want to pollute the issues area, but I'd like to say thanks to chris2511 for this software. I used to use TinyCA but it's showing its age. XCA is simple, easy and reliable - and congratulations for keeping up up to date for over ten years!

Well done!

**Comment by xPucTu4** [comment]
Nice idea to open issue for that. I remember sending him email many months ago, but I will repeat myself here:  
Thanks for this great software!  💋

**Comment by n9yty** [comment]
I emailed as well, curious if he would take a donation... Anything set up for that?  He deserves it!

**Comment by chris2511** [comment]
In 2015 with commit e1874e13bbe310600dd9ea7d8a996fd56f7046cf I stopped begging for donations, because the earnings were too small.
However, donations are very welcome at my paypal account: christian@hohnstaedt.de

**Comment by 15knots** [comment]
I have been using XCA 1.3.2  for some years now and I am glad to see this is still/again maintained.
@chris2511 If you are interested in packaging this as rpms for various Linux distros, I would be glad to help.

**Comment by vitomeuli** [comment]
Great piece of software! Thanks a lot!

**Comment by jonhp** [comment]
I would also like to thank @chris2511 for his great XCA application. Most appreciated.
